pandas/core/frame.py

```diff
@@ -8743,6 +8743,20 @@ def aggregate(self, func=None, axis: Axis = 0, *args, **kwargs):
 
     agg = aggregate
 
+    @deprecate_nonkeyword_arguments(version=None, allowed_args=["self"])
+    @doc(NDFrame.any, **_shared_doc_kwargs)
+    def any(
+        self: DataFrame,
```

Comment:
Based on https://github.com/python/mypy/issues/11759#issuecomment-1084690934 it is possible to just have the type annotations without the actual implementation. 

(Hope it doesn't break the doc generation.)

Reply:
Nice, does this work (using `...`)? I thought somethling like `any : Callable[...]  # or Probably a protocol` but that will then not allow you to change the doc-string

Reply:
Not sure actually - @yadav-sachin shall we just remove any modifications to `frame.py` and `series.py` in this PR, to keep it focused (and to get it through)? Apologies for contradicting requests from my part

Reply:
> Nice, does this work (using `...`)? I thought somethling like `any : Callable[...] # or Probably a protocol` but that will then not allow you to change the doc-string

When checking with mypy, I was not getting any errors with this method (using `...`)

Reply:
> When checking with mypy, I was not getting any errors with this method (using `...`)

I meant more whether this works in practice. I would expect that any() wouldn't do anything when called on Series/DataFrame (unless `...` magically delegates to NDFrame - would be cool if it does).